desktop/core/src/desktop/lib/thrift_util_test.py

```diff
@@ -343,5 +347,41 @@ def test_wrapper_with_retry(self):
       # Could check output for several "Thrift exception; retrying: some error"
 
 
+class TestThriftJWT(unittest.TestCase):
+  def setUp(self):
+    self.sample_token = "some_jwt_token"
+
+    self.client = make_logged_in_client(username="test_user", groupname="default", recreate=True, is_superuser=False)
+    self.user = rewrite_user(User.objects.get(username="test_user"))
+
+    self.user.profile.update_data({'jwt_access_token': self.sample_token})
+    self.user.profile.save()
```

Comment:
@romainr  I am creating a user and rewriting it, but when trying to find the same user in thrift_util.py to fetch the token it is not able to find its profile (which gets set from rewrite_user). Any insights of where I have missed something? Debugged and tried a lot of things but no progress as to where am I missing something.

```
Creating test database for alias 'default'...
[08/Sep/2021 02:35:07 -0700] models       INFO     HuePermissions: 26 added, 0 updated, 0 up to date, 0 stale, 0 deleted
[08/Sep/2021 02:35:07 -0700] backend      INFO     Augmenting users with class: <class 'desktop.auth.backend.DefaultUserAugmentor'>
[08/Sep/2021 02:35:07 -0700] attempts     DEBUG    AXES: Skipping clean for expired access attempts because no AXES_COOLOFF_TIME is configured
[08/Sep/2021 02:35:07 -0700] database     INFO     AXES: Successful login by {username: "test_user", ip_address: "None", user_agent: "<unknown>", path_info: "<unknown>"}.
[08/Sep/2021 02:35:07 -0700] backend      DEBUG    Found user test_user
E
======================================================================
ERROR: test_jwt_injection_thrift (desktop.lib.thrift_util_test.TestThriftJWT)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/harsh.gupta/Workspace/upstream/hue/desktop/core/src/desktop/lib/thrift_util_test.py", line 380, in test_jwt_injection_thrift
    service, protocol, transport = thrift_util.connect_to_thrift(conf)
  File "/Users/harsh.gupta/Workspace/upstream/hue/desktop/core/src/desktop/lib/thrift_util.py", line 351, in connect_to_thrift
    elif user.profile.data.get('jwt_access_token'):
AttributeError: 'User' object has no attribute 'profile'
-------------------- >> begin captured logging << --------------------
useradmin.models: INFO: HuePermissions: 26 added, 0 updated, 0 up to date, 0 stale, 0 deleted
desktop.auth.backend: INFO: Augmenting users with class: <class 'desktop.auth.backend.DefaultUserAugmentor'>
axes.attempts: DEBUG: AXES: Skipping clean for expired access attempts because no AXES_COOLOFF_TIME is configured
axes.handlers.database: INFO: AXES: Successful login by {username: "test_user", ip_address: "None", user_agent: "<unknown>", path_info: "<unknown>"}.
desktop.auth.backend: DEBUG: Found user test_user
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 1 test in 0.258s

FAILED (errors=1)
Destroying test database for alias 'default'...
[08/Sep/2021 02:35:07 -0700] test         INFO     Tests (desktop.lib.thrift_util_test:TestThriftJWT) returned 1
harsh.gupta@harsh hue % 
```